messagequeue/chunker.go

```diff
@@ -3,72 +3,36 @@ package messagequeue
 import (
 	"sync"
 	"sync/atomic"
-	"time"
 
 	"github.com/beaconsoftwarellc/gadget/v2/errors"
 
 	"github.com/beaconsoftwarellc/gadget/v2/timeutil"
 )
 
 const (
-	stateStopped                     = 0
-	stateRunning                     = 1
-	defaultChunkSize   uint32        = 10
-	defaultEntryExpiry time.Duration = 10 * time.Second
-	minimumExpiry                    = time.Millisecond
-	maximumExpiry                    = 24 * time.Hour
-	minimumChunkSize                 = 1
-	maximumChunkSize                 = 1024
+	stateStopped = 0
```

Comment:
https://github.com/beaconsoftwarellc/gadget/pull/43/files updates these, should that PR be canceled if the related code is just being removed?

Reply:
Yeah, I was thinking that as well. I was trying to keep the PR's relatively small though. The base for this PR is the other one's branch so it is kinda building.